README.adoc

```diff
@@ -712,4 +714,136 @@ added after the original pull request but before a merge.
   other target branch in the main project).
 * When writing a commit message please follow https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html[these conventions],
   if you are fixing an existing issue please add `Fixes gh-XXXX` at the end of the commit
-  message (where XXXX is the issue number).
\ No newline at end of file
+  message (where XXXX is the issue number).
```

Comment:
Where are these changes from? Looks like something from a newer version.

Reply:
these might have changed after cloning the newer version of forked repo, do you want me to revert these back?

Reply:
yes